### DISCUSSION TITLE
docs flutter

### DISCUSSION THREAD

**Discussion by freddie-valenzuela** [zealdocs/zeal#1388] [Q&A] [answered]
Buenas tardes, todos los programadores de esta palicacion son muy buenos, de mi trabajo estoy buscando la documentacion de flutter, veo que aun no esta disponible, es posible saber si la van a implementar?

Gracias por su buen trabajo

**Comment by trollixx** [comment, marked as answer]
Flutter project provides their own docsets with feeds:

* https://api.flutter.dev/offline/flutter.xml
* https://master-api.flutter.dev/offline/flutter.xml
